src/sentry/static/sentry/app/views/organizationIntegrations/IntegrationDirectorySentryAppRow.tsx

```diff
@@ -126,7 +126,7 @@ const color = {
 
 type StatusIndicatorProps = {status: string; theme?: any};
 
-const StatusIndicator = styled(({status, ...props}: StatusIndicatorProps) => {
+export const StatusIndicator = styled(({status, ...props}: StatusIndicatorProps) => {
```

Comment:
@NisanthanNanthakumar I think we should just put the `StatusIndicator` in its own file

Reply:
Actually, I think the ideal solution would be to make a component that could be used for all types of integrations and put that into its own file

Reply:
@scefali we don't currently use StatusIndicator in any other components